platform/jewel/ui/src/main/kotlin/org/jetbrains/jewel/ui/component/styling/SearchMatchStyle.kt

```diff
@@ -0,0 +1,71 @@
+// Copyright 2000-2025 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license.
+package org.jetbrains.jewel.ui.component.styling
+
+import androidx.compose.runtime.ProvidableCompositionLocal
+import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.graphics.Color
+import org.jetbrains.jewel.foundation.GenerateDataFunctions
+
+@GenerateDataFunctions
+public class SearchMatchStyle(public val colors: SearchMatchColors) {
```

Comment:
We don't seem to be accounting for it yet, but it seems like we should include the metrics too, since we should if not now at least in a future release draw the highlight by ourselves. According to `com.intellij.util.ui.UIUtil#drawSearchMatch(java.awt.Graphics2D, float, float, int, java.awt.Color, java.awt.Color)`, there are:
 * A padding compared to the line height: `PaddingValues(vertical = 2.dp)`
 * A corner radius of `2.5.dp`

We need to check that using these values in Jewel produces the expected results compared to the Swing alternative, and in case tweak them accordingly, but in any case we should have them already to spare us the deprecation cycle in the (near?) future.